cbmc/proofs/IotMqtt_SubscribeAsync/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
```

Comment:
Should we move to installing the empty cbmc-batch.yamls and letting CI regenerate them?  Maybe check with Jonathan or Kareem on the status of this transition.

Reply:
We currently have a task to address this (on a separate PR). I updated the `cbmc-batch.yaml` file to match our Makefile standard.